app/javascript/components/IssuesCategoriesChart.vue

```diff
@@ -27,7 +27,7 @@ export default {
 
       switch (this.title.toLowerCase()) {
         case 'suspension':
-          complianceType = 'trade'

```

Comment:
This switch statement doesn't seem to do anything anymore?
 Does it just check that the title is one of the three allowed values - if so, could this be expressed more clearly?